Update 06_Mobile_access.md (#4325)
FreshRSS/FreshRSS: docs/en/users/06_Mobile_access.md

1. Under the section “Profile”, click on the link like `https://rss.example.net/api/` next to the field “API password”.
2. Click on first link “Check full server configuration”:
	* If you get *PASS* then you are done; all is well.
	* If you get *Bad Request!* or *Not Found*, then your server probably does not accept slashes `/` that are escaped `%2F`, see the next section "Fix server configuration".

> 	* If you receive any other error message, see the next section "Fix server configuration".

Reviewer: ```suggestion
	* If you receive any other error message, see the next section “Fix server configuration”.
```

Either typographic quotes or *`*`emphasis`*`*

### Fix server configuration

* Click on the second link “Check partial server configuration (without `%2F` support)”:
	* If you get `PASS`, then the problem is indeed that your server does not accept slashes `/` that are escaped `%2F`.
		* With Apache, remember the directive [`AllowEncodedSlashes On`](http://httpd.apache.org/docs/trunk/mod/core.html#allowencodedslashes)
		* Or use a client that does not escape slashes (such as EasyRSS) ([`check client list`](https://github.com/FreshRSS/FreshRSS#apis--native-apps)).

> 	* If you get *Service Unavailable!*, then check the preceding section "Enable the API in FreshRSS" again.

Reviewer: ```suggestion
	* If you get *Service Unavailable!*, then check the preceding section “Enable the API in FreshRSS” again.
```

1. Under the section “Profile”, click on the link like `https://rss.example.net/api/` next to the field “API password”.
2. Click on first link “Check full server configuration”:

> 	* If you get *PASS* then you are done, all is good.

Reviewer 2: ```suggestion
	* If you get *PASS* then you are done; all is well.
```

> 	* If you get *Bad Request!* or *Not Found*, then your server probably does not accept slashes `/` that are escaped `%2F`, see next chapter "Fix server configuration".
> 	* If you get any other error message, see next chapter "Fix server configuration".
> 

Reviewer 2: ```suggestion
	* If you get *Bad Request!* or *Not Found*, then your server probably does not accept slashes `/` that are escaped `%2F`, see the next section "Fix server configuration".
	* If you receive any other error message, see the next section "Fix server configuration".
```

### Fix server configuration

* Click on the second link “Check partial server configuration (without `%2F` support)”:
	* If you get `PASS`, then the problem is indeed that your server does not accept slashes `/` that are escaped `%2F`.
		* With Apache, remember the directive [`AllowEncodedSlashes On`](http://httpd.apache.org/docs/trunk/mod/core.html#allowencodedslashes)
		* Or use a client that does not escape slashes (such as EasyRSS) ([`check client list`](https://github.com/FreshRSS/FreshRSS#apis--native-apps)).

> 	* If you get *Service Unavailable!*, then check from above chapter "Enable the API in FreshRSS" again.

Reviewer 2: ```suggestion
	* If you get *Service Unavailable!*, then check the preceding section "Enable the API in FreshRSS" again.
```

		* If you get *File not found!*, check your server `fastcgi_split_path_info`.
	* If you get *FAIL 64-bit or GMP extension!*, then your PHP version does not pass the requirement of being 64-bit and/or have PHP [GMP](http://php.net/gmp) extension.
		* The easiest is to add the GMP extension. On Debian / Ubuntu: `sudo apt install php-gmp`

> 	* Update and try again from the preceding section "Testing".

Reviewer: ```suggestion
	* Update and try again from the preceding section “Testing”.
```

Minor quotes

		* If you get *File not found!*, check your server `fastcgi_split_path_info`.
	* If you get *FAIL 64-bit or GMP extension!*, then your PHP version does not pass the requirement of being 64-bit and/or have PHP [GMP](http://php.net/gmp) extension.
		* The easiest is to add the GMP extension. On Debian / Ubuntu: `sudo apt install php-gmp`

> 	* Update and try again from above chapter "Testing".

Reviewer 2: ```suggestion
	* Update and try again from the preceding section "Testing".
```
